### PR TITLE
Fix chalk import in plugins

### PR DESCRIPTION
**Pull request by Jeymisson**

### CODE REVIEW

**Review by valentino-amadeus — APPROVED** [review]
LGTM

It seems that you'd want to apply the same change [here](https://github.com/vtex/cli-plugin-plugins/blob/d5cb9f53596ee808912212fddc5005320625f530/src/commands/plugins/uninstall.ts#L3).

**Review by williamads — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## [Unreleased]
 
+## [1.13.1] - 2021-05-25
+
+### Fixed
+- fix typo when importing `chalk`
```
Comment (gris) [inline CHANGELOG.md:6]:
```suggestion
- Typo when importing `chalk`
```

**Review by gris — APPROVED** [review]
Seems good to implement the same for uninstall, like @valentino-amadeus suggested. May I ask you to follow up with this @williamads ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## [Unreleased]
 
+### Fixed
+- Typo when importing `chalk`
+
```
Comment (valentino-amadeus) [inline CHANGELOG.md:5]:
```suggestion
```

**Review by valentino-amadeus — COMMENTED** [review]
This PR actually refers to the `## [1.13.1] - 2021-05-25` section (see Jey's second commit).

File: CHANGELOG.md (hunk 2 above)
Comment (williamads) [reply CHANGELOG.md:5]: Nice point, thanks! `## [1.13.1] - 2021-05-25` was released before the finish of the pull request. I'm planning to do a new release to include new changes on NPM. Without add this change, there will be a blank release on the repo. Make sense for you?

Comment (valentino-amadeus) [reply CHANGELOG.md:5]: Yeah, the tag has indeed been created before it should've been. You'll have to make a new release, in fact. Make sure you specify in the CHANGELOG, under the section `## [1.13.1] - 2021-05-25`, the changes that have already been applied. Then you can announce the new changes in a new section.

**Pull request merged**